test/jaxp/javax/xml/jaxp/unittest/common/prettyprint/PrettyPrintTest.java

```diff
@@ -180,6 +178,53 @@
         };
     }
 
+    private final String xml1 = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>"
+            + "<a><b>element b</b><c>element c</c><d xml:space=\"preserve\">TRUE</d><e>test</e></a>";
+    private final String expected1 = "<a>\n"
+            + "    <b>element b</b>\n"
+            + "    <c>element c</c>\n"
+            + "    <d xml:space=\"preserve\">TRUE</d>\n"
+            + "    <e>test</e>\n"
+            + "</a>\n";
+
+    private final String xml2 = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>"
+            + "<l0><l1><l2 xml:space=\"preserve\">level 2</l2> level 1 <l2>level 2</l2></l1>"
+            + "<l1 xml:space=\"preserve\"><l2>level 2</l2> level 1 <l2>level 2</l2></l1></l0>";
+    private final String expected2 = "<l0>\n"
+            + "    <l1>\n"
+            + "        <l2 xml:space=\"preserve\">level 2</l2>\n"
+            + "         level 1 \n"
+            + "        <l2>level 2</l2>\n"
+            + "    </l1>\n"
+            + "    <l1 xml:space=\"preserve\"><l2>level 2</l2> level 1 <l2>level 2</l2></l1>\n"
+            + "</l0>\n";
+
+    /*
+     * Bug: 8265073
+     * source and expected output
+     */
+    @DataProvider(name = "preserveSpace")
+    public Object[][] preserveSpace() throws Exception {
+        return new Object[][]{
+            {xml1, expected1},
+            {xml2, expected2},
+        };
+    }
+
+    /**
+     * Bug: 8265073
+     * Verifies that the scope of the preserve attribute is applied properly
+     * within the relevant elements.
+     * @param xml the source
+     * @param expected the expected result
+     * @throws Exception if the test fails
```

Comment:
`Exception` is thrown from `transform` method. So the description is not wrong, but it may imply the objective of this test failed, i.e. the result of `assertEquals()`.